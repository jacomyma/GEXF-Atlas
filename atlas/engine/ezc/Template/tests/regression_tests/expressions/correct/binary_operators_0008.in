{var $foo = 42, $bar = 101, $bar2 = 201, $arr = array( 'foo' => 401 )}
0225:[{debug_dump( ( $foo == 5 ) + 5 )}]
0226:[{debug_dump( ( $foo == 5 ) + -$foo )}]
0227:[{debug_dump( ( $foo == 5 ) - 5 )}]
0228:[{debug_dump( ( $foo == 5 ) - -$foo )}]
0229:[{debug_dump( ( $foo == 5 ) * 5 )}]
0230:[{debug_dump( ( $foo == 5 ) * -$foo )}]
0231:[{str_number( ( $foo == 5 ) / 5 , 2, ',', '' )}]
0232:[{str_number( ( $foo == 5 ) / -$foo , 2, ',', '' )}]
0233:[{debug_dump( ( $foo == 5 ) % 5 )}]
0234:[{debug_dump( ( $foo == 5 ) % -$foo )}]
0235:[{debug_dump( ( $foo == 5 ) == 5 )}]
0236:[{debug_dump( ( $foo == 5 ) == -$foo )}]
0237:[{debug_dump( ( $foo == 5 ) === 5 )}]
0238:[{debug_dump( ( $foo == 5 ) === -$foo )}]
0239:[{debug_dump( ( $foo == 5 ) != 5 )}]
0240:[{debug_dump( ( $foo == 5 ) != -$foo )}]
0241:[{debug_dump( ( $foo == 5 ) !== 5 )}]
0242:[{debug_dump( ( $foo == 5 ) !== -$foo )}]
0243:[{debug_dump( ( $foo == 5 ) < 5 )}]
0244:[{debug_dump( ( $foo == 5 ) < -$foo )}]
0245:[{debug_dump( ( $foo == 5 ) <= 5 )}]
0246:[{debug_dump( ( $foo == 5 ) <= -$foo )}]
0247:[{debug_dump( ( $foo == 5 ) > 5 )}]
0248:[{debug_dump( ( $foo == 5 ) > -$foo )}]
0249:[{debug_dump( ( $foo == 5 ) >= 5 )}]
0250:[{debug_dump( ( $foo == 5 ) >= -$foo )}]
0251:[{debug_dump( ( $foo == 5 ) && 5 )}]
0252:[{debug_dump( ( $foo == 5 ) && -$foo )}]
0253:[{debug_dump( ( $foo == 5 ) || 5 )}]
0254:[{debug_dump( ( $foo == 5 ) || -$foo )}]
0255:[{debug_dump( ( $foo == 5 ) . 5 )}]
0256:[{debug_dump( ( $foo == 5 ) . -$foo )}]
