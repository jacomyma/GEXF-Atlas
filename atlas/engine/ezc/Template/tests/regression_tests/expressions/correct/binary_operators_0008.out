0225:[5]
0226:[-42]
0227:[-5]
0228:[42]
0229:[0]
0230:[0]
0231:[0,00]
0232:[0,00]
0233:[0]
0234:[0]
0235:[false]
0236:[false]
0237:[false]
0238:[false]
0239:[true]
0240:[true]
0241:[true]
0242:[true]
0243:[true]
0244:[true]
0245:[true]
0246:[true]
0247:[false]
0248:[false]
0249:[false]
0250:[false]
0251:[false]
0252:[false]
0253:[true]
0254:[true]
0255:['5']
0256:['-42']
